content/src/main/content/jcr_root/apps/acs-commons/touchui-widgets/source/touchui-limit-parsys.js

```diff
@@ -23,6 +23,7 @@
  * eg. to limit the components to 4 on rightpar of /content/geometrixx/en.html
  * set acsComponentsLimit=4 on /etc/designs/geometrixx/jcr:content/homepage/rightpar
  */
+ window.Granite.author = window.Granite.author || {};
```

Comment:
@ahmed-musallam i think it still breaks in extendComponentDrop() with gAuthor.edit.actions undefined

this extension should be loaded after Granite.author is available, otherwise its unusable